yarn.lock

```diff
@@ -3986,8 +3986,6 @@ eslint-plugin-html@^6.0.0:
 
 "eslint-plugin-oppia@file:scripts/linters/custom_eslint_checks":
   version "0.0.0"
-  dependencies:
-    requireindex "~1.1.0"
```

Comment:
Undo these changes! [Why is this happening automatically?]

Reply:
We need to this in at least one PR because in the first PR there was a separate package.json file for eslint-checks which then removed on a later stage, but I didn't updated the yarn file. So we need to do it in one of the new PRs

Reply:
Can you please create an issue to find whether we can pass eslint plugins directly instead of using package.json file? [I think this will help us avoid auto-update packages because of eslint.]

Also, I'm fine with these changes for this PR!

Reply:
I've created the issue https://github.com/oppia/oppia/issues/10551